packages/admin/blocks-admin/src/blocks/helpers/createCompositeBlockTextField.tsx

```diff
@@ -0,0 +1,27 @@
+import { TextField, TextFieldProps } from "@comet/admin";
+import * as React from "react";
+import { FormProps } from "react-final-form";
+
+import { BlocksFinalForm } from "../../form/BlocksFinalForm";
+import { createCompositeSetting } from "./composeBlocks/createCompositeSetting";
+
+interface Options {
+    defaultValue: string;
+    formProps?: FormProps<Record<"value", string>, Record<"value", string>>;
```

Comment:
Do you have a specific use case in mind for this option?

Reply:
no specific one but it enables the possibility to add validators etc.

Reply:
Validation would be on field level though 🤔

Reply:
ah right. hm yeah are there any other properties of the form props that devs need to set?

Reply:
I don't think so. Maybe remove it for now and add it later if we really need it?